docs/resources/modelarts_resource_pool.md

```diff
@@ -24,62 +26,205 @@ resource "huaweicloud_modelarts_resource_pool" "test" {
+
+  Changing this parameter will create a new resource.
+
+* `workspace_id` - (Optional, String, ForceNew) Specifies the workspace ID, which defaults to **0**.
 
   Changing this parameter will create a new resource.
 
-* `workspace_id` - (Optional, String, ForceNew) Workspace ID, which defaults to 0.  
+* `description` - (Optional, String) Specifies the description of the dedicated resource pool.  
 
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the resource pool.
+  Valid values are **prePaid** and **postPaid**, defaults to **postPaid**.
   Changing this parameter will create a new resource.
 
-* `description` - (Optional, String) The description of the dedicated resource pool.  
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the resource pool.
+  Valid values are **month** and **year**. This parameter is mandatory if `charging_mode` is set to **prePaid**.
+  Changing this parameter will create a new resource.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the resource pool.
+  If `period_unit` is set to **month**, the value ranges from **1** to **9**.
+  If `period_unit` is set to **year**, the value ranges from **1** to **3**.
+  This parameter is mandatory if `charging_mode` is set to **prePaid**.
+  Changing this parameter will create a new resource.
+
+* `auto_renew` - (Optional, String) Specifies whether auto-renew is enabled. Valid values are **true** and **false**.
+
+-> **NOTE:** `charging_mode`, `period_unit`, `period`, `auto_renew` must be specified when `vpc_id` is specified.
 
 <a name="ModelartsResourcePool_ResourceFlavor"></a>
 The `resources` block supports:
 
-* `flavor_id` - (Required, String) The resource flavor ID.  
+* `flavor_id` - (Required, String) Specifies the resource flavor ID.  
+
+* `count` - (Required, Int) Specifies the number of resources of the corresponding flavors.
+
+* `node_pool` - (Optional, String) Specifies the name of resource pool nodes. It can contain **1** to **50**
+  characters, and should start with a letter and ending with a letter or digit, only lowercase letters, digits,
+  hyphens (-) are allowed, and cannot end with a hyphen (-).
+
+* `max_count` - (Optional, Int) Specifies the max number of resources of the corresponding flavors.
 
-* `count` - (Required, Int) Number of resources of the corresponding flavors.
+* `vpc_id` - (Optional, String) Specifies the VPC ID. It is mandatory when `resources.subnet_id`,
+  `resources.security_group_ids` is specified, and can not be specified if `network_id` is specified.
 
-* `azs` - (Optional, List) AZs for resource pool nodes.
-  The [azs](#ModelartsResourcePool_ResourceFlavor_azs) structure is documented below.
+* `subnet_id` - (Optional, String) Specifies the network ID of a subnet. It is mandatory when
+  `resources.security_group_ids`is specified, and can not be specified if `network_id` is specified.
```

Comment:
is `vpc_id` required for this parameter?

Reply:
yes, vpc_id is required if this param is set